app/controllers/master/api/services_controller.rb

```diff
@@ -31,10 +31,10 @@ def destroy
   private
 
   def service
-    @service ||= provider.accessible_services.find(params[:id])
+    @service ||= provider.accessible_services.find(params.permit!.to_h[:id])
```

Comment:
Why not

```suggestion
    @service ||= provider.accessible_services.find(params.require(:id))
```

Reply:
@hallelujah I preferred permit because what if we reuire id and its missing so may be can break the code.

Reply:
params.require raises an exception if it is not present.

As you commented on one of my PR

Reply:
In this case all usages of `#service` appear to expect it NOT to be `nil`. That's why if we raise early when parameter is missing, that makes more sense than leaving the error for later. And the error message will be more meaningful.

Reply:
I think that also API documentation can be useful to see what parameters we require. If parameter is required by documentation, in most cases we should better require it in code as well.

Reply:
@akostadinov checking API documentation is a good option.